tests/ui/static/static_sized_requirement.rs

```diff
@@ -1,11 +1,12 @@
-//@ build-pass (FIXME(62277): could be check-pass?)
+//@ add-core-stubs
+//@ check-pass
```

Comment:
Question: hm, I wonder if this test already `check-pass` before, why does it need to `build-pass` before 🤔

Reply:
I'm not sure? Given that it had the `FIXME`, I figured it was worth changing it if it passed when I did.

Reply:
Sure, that seems fine.